doc/src/elasticsearch.md

```diff
@@ -131,6 +131,12 @@ Nodes should be upgraded one at a time to ensure continous operation of the
 cluster. Upgrading nodes is done by changing the role of the machine to
 *elasticsearch7*.
 
+### Upgrade/Migration to OpenSearch
+
```

Comment:
mention that ES is unsupported on newer platforms and that it must be migrated to OS before upgrading the system.